common/vm_get_vmtools_status.yml

```diff
@@ -58,10 +51,12 @@
     - name: Set fact of VMware Tools running status
       set_fact:
         vmtools_is_running: "{{ get_vmtools_info_retry.vmtools_info.vm_tools_running_status == 'guestToolsRunning' }}"
-  when: vmtools_is_installed and vmtools_not_ready
+  when:
+    - vmtools_is_installed | default(False)
+    - vmtools_not_ready | default(True)
 
 - debug:
     msg:
-      - "VMware tools is installed in guest: {{ vmtools_is_installed }}"
-      - "VMware tools is running in guest: {{ vmtools_is_running }}"
-      - "VMware tools is installed as open-vm-tools: {{ vmtools_is_ovt }}"
+      - "VMware tools is installed in guest: {{ vmtools_is_installed | default(False) }}"
```

Comment:
Do we need to add checking these 3 variables defined or not in linux or windows test setup and test cases?